hawk/lib/crm_script.rb

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) 2015 Kristoffer Gronlund <kgronlund@suse.com>
+# See COPYING for license information.
+
+require 'pty'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.